src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -306,25 +151,23 @@ AudioDataHandle FFmpegAudioFileCache::Open_File(const Utf8String &filename)
     FFmpegOpenAudioFile open_audio;
```

Comment:
I was taking the naming from `MilesAudioFileCache`. Should i still change?

Reply:
Up to you. `open_audio` reads odd, because what is open about this audio? Would there be a `close_audio`? So I think `audio_file` would be a better name.